server/service/testing_utils.go

```diff
@@ -630,6 +631,7 @@ func mdmConfigurationRequiredEndpoints() []struct {
 		{"POST", "/api/latest/fleet/device/%s/migrate_mdm", true, true},
 		{"POST", "/api/latest/fleet/mdm/apple/profiles/preassign", false, true},
 		{"POST", "/api/latest/fleet/mdm/apple/profiles/match", false, true},
+		{"POST", "/api/latest/fleet/mdm/commands/run", false, false},
```

Comment:
I think I forgot to add the new `GET mdm/commandresults` endpoint here. Looks like there may be others missing too.